examples/pstore/pscli.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"storj.io/storj/pkg/pstore"
+	"github.com/urfave/cli"
+	"log"
+	"os"
+	"sort"
+)
+
+type argError struct {
```

Comment:
also still the comment about github.com/zeebo/errs making this easier

Reply:
Thank goodness for this library